ocpp/charge_point.py

```diff
@@ -356,10 +356,25 @@ async def _handle_call(self, msg):
             call_unique_id_required = "call_unique_id" in handler_signature.parameters
             # call_unique_id should be passed as kwarg only if is defined explicitly
             # in the handler signature
+            inject_response = getattr(handler, "_inject_response", False)
             if call_unique_id_required:
-                response = handler(**snake_case_payload, call_unique_id=msg.unique_id)
+                if inject_response:
+                    response = handler(
+                        **snake_case_payload,
+                        call_unique_id=msg.unique_id,
+                        on_response=response_payload,
```

Comment:
It would be better the term "call_response". The "on" prefix is given to functions/callbacks as an action will be performed via that callback function. Here we want to pass an argument..